src/Uno.Extensions.Templates/content/unoapp-extensions/build/steps/wasm.yml

```diff
@@ -0,0 +1,71 @@
+parameters:
+- name: major
+  type: string
+  default: '0'  
+- name: minor
+  type: string
+  default: '0'
+- name: package_id
+  type: string
+  default: 'release_Wasm'
+- name: project_to_build
+  type: string
+  default: ''
+- name: project_folder
```

Comment:
We should be able to infer project_folder from the project_to_build filename. We can leave this as a parameter in case it needs to be overridden but if the value is '' we should infer the value from the project filename